concepts/05 UI Components/DataGrid/70 Custom Keyboard Navigation/10 Focus Cells Programatically.md

```diff
@@ -0,0 +1,7 @@
+You can use the following methods to focus a specific cell:
+
+- [focus](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#focuselement)     
+Allows you to set focus on a cell. You can use the [getCellElement](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#getCellElementrowIndex_dataField) method to get elements of required cells.
```

Comment:
```suggestion
Allows you to move focus to a cell. You can call the [getCellElement](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#getCellElementrowIndex_dataField) method to get elements of required cells.
```

P.S. What connection the mentioned getCellElement method has with the focus movement scenario we describe?

Reply:
You call getCellElement to get a cell container so you can then call focus.
Like
`const cell = e.getCellElement();
focus(cell);`

Reply:
Is it a mandatory or optional step? If it's optional, then this wording is probably fine as it is. Otherwise, we should state that one should call the mentioned methods in conjunction in order to achieve the desired result.

From what I gather, judging from your example, this is a mandatory step. Therefore, you first obtain this object/entity named "cell" whatever it is -- and then you pass it to the `focus` method as a parameter. Shouldn't we explain things here exactly as they are?

Reply:
I updated this part